src/compiler/watch.ts

```diff
@@ -329,8 +329,8 @@ function createTabularErrorsDisplay(filesInError: (ReportFileInError | undefined
 }
 
 /** @internal */
-export function isBuilderProgram(program: Program | BuilderProgram): program is BuilderProgram {
-    return !!(program as BuilderProgram).getState;
+export function isBuilderProgram<T extends BuilderProgram>(program: Program | BuilderProgram): program is T {

```

Comment:
This seems unsafe; this could conform to anything, right?

Reply:
I guess it's copied from somewhere else...

Reply:
Yep